docs/README.md

```diff
@@ -1,39 +1,58 @@
-# Docs
+# Kyma Community Modules
 
 ## Overview
 
-The `docs` folder contains two subfolders - `user` and `contributor`. 
+The community modules are modules provided by the Kyma community and can be installed alongside the managed modules. They are not automatically updated or maintained, but they can be used to enhance your Kyma experience with additional features and capabilities.
```

Comment:
```suggestion
Community modules are modules provided by the Kyma community. Use them to enhance your Kyma experience with additional features and capabilities. You can install community modules alongside managed modules. Unlike managed modules, community modules aren't automatically updated or maintained.
```

Should we also add a disclaimer here about losing SLA if you're a managed Kyma user? For example, "In SAP BTP, Kyma runtime community modules aren't subject to the Service Level Agreement (SLA)."?